### ISSUE TITLE
JSPM issue with `#` characters in filenames (because of es5-ext dependency)

### ISSUE THREAD

**Issue by mazniak** [jspm/jspm-cli#1147]
I'm using JSPM 0.16.6 and SystemJS 0.19.0. In an empty, standard JSPM project I ran `jspm install ohm=npm:ohm-js` and created a file — `wat.js` — with the contents `require('ohm');`

Running `jspm run wat` produces

```
err  Error: ENOENT, open '/private/tmp/foo/jspm_packages/npm/es5-ext@0.10.7/string/%23/contains.js'
         at Error (native)
	Error loading file:///private/tmp/foo/jspm_packages/npm/es5-ext@0.10.7/string/#/contains.js as "es5-ext/string/#/contains" from file:///private/tmp/foo/jspm_packages/npm/d@0.1.1/index.js
```

This is referenced in other issues as fixed, so I'm not sure if this is user error or what.

**Comment by guybedford** [comment]
@mazniak thanks, yes it's more complex than it seems. Hopefully have it right this time.

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/systemjs/commit/9cfd46a086449217146d49234ecf28c1ca5f3074.

**Comment by guybedford** [comment]
Nope, wasn't that simple either, fixed in https://github.com/systemjs/systemjs/commit/e06ed03013cf9f6ec25acf3bca263299f69d0b4a.

**Comment by teohhanhui** [comment]
@guybedford Are you tagging new releases considering the critical nature of this bug? It makes the whole thing completely unusable.

**Comment by guybedford** [comment]
Released in 0.16.7.
